frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -28,13 +28,12 @@
           :title="input.question"
           :optionId="`${input.name}_0`"
           :placeholder="input.placeholder"
-          :initialOptions="input.options[0]"
+          :initialText="input.options[0].text"
```

Comment:
Normally this prop is called `value` (See an example here: https://v2.vuetifyjs.com/en/api/v-textarea/#props)